release-notes.txt

```diff
@@ -1,3 +1,622 @@
+RIOT-2021.01 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release:
+===================
+
+The 2021.01 release includes:
+ - Device Firmware Upgrade support for USBUS
+ - USB HID minimal implementation
+ - IEEE 802.15.4 security initial implementation
+ - 6LowPAN Minimal Forwarding and Selective Fragment Recovery initial
+   implementation
+ - Initial support for QN808x, SAMD10 STM32MP1 and STM32L5 CPUs
+ - Initial support for Cortex-M33
+ - A module for selecting entropy sources
+ - Initial support for the Decawave Impulse Radio-Ultra Wideband (UWB Core)
+ - Capability to join or leave link layer multicast groups (GNRC)
+ - Support for IN865 region in GNRC LoRaWAN
+ - 83 additional modules supported by Kconfig
+ - 14 new boards, 6 new device drivers, 8 packages updated
+
+[UPDATE!!!]
+532 pull requests, composed of 1439 commits, have been merged since the
+last release, and 65 issues have been solved. XXX people contributed with
+code in 114 days. 3852 files have been touched with 348615 (+) insertions and
+52513 deletions (-).
+
+Notations used below:
+=====================
+
+    + means new feature/item
+    * means modified feature/item
+    - means removed feature/item
+
+New features and changes
+========================
+
+Core
+----
++ msg_bus: allow to differ between messages form bus and from thread
++ mutex: Add mutex_cancel
++ mbox: add mbox_size(), mbox_avail()
+* mutex: compatibility with non-C languages
+* refactor thread info 
+- remove kernel_types.h
+
+System Libraries
+----------------
++ Add thread-safe Malloc
++ sys/tsrb: Make thread-safe
++ mtd_mapper: implement pagewise functions
++ xtimer: introduce xtimer_msleep()
++ usbus: Add USB peripheral serial string support
++ hashes: Add PBKDF2-sha256 implementation
++ usbus/dfu: add Device Firmware Upgrade support for USBUS
```

Comment:
```suggestion
+ usbus/dfu: add Device Firmware Upgrade support for USBUS
+ USB HID minimal implementation
```

Reply:
this was indeed added, but in Device Drivers. The original PR had the "Area: drivers" tag (see https://github.com/RIOT-OS/RIOT/pull/14629). I will keep this one then

Reply:
check 02dd998decdcb84a1f003f29d1d3cd8c982bfef5